action.yml

```diff
@@ -0,0 +1,105 @@
+name: Dispatch CI
+description: GitHub Action to trigger ci.yml workflow_dispatch event via GitHub API
+
+runs:
+  using: composite
+  steps:
+
+    - name: Get type
+      id: gettype
+      shell: bash
+      env:
+        GITHUB_REPOSITORY: ${{ github.repository }}
+      run: |
+        # repository that has a twice-weekly schedule run on consequtive days - use day of week
+        date_part=$(date +%u)
+        type=""
+        if [[ $GITHUB_REPOSITORY == "silverstripe/installer" ]] || \
+            [[ $GITHUB_REPOSITORY == "silverstripe/recipe-kitchen-sink" ]]
+        then
+          # respository that has daily schedule - use hour
+          date_part=$(date +%H)
+        fi
+        if (( $date_part % 2 == 1 )); then
+          # latest major, next-minor e.g. "5"
+          type=latest,next-minor
```

Comment:
next-minor doesn't seem to ever be used? What is the intention there?

Reply:
I needed something for minor branch so that `minor_type=$(echo $TYPE | cut -d "," -f 2)` below doesn't fail.  `next-minor` is what's used on https://emtek.net.nz/rhino/tables?t=releases so I just reused it here.  `latest` (major) and `next-minor` are synonomous, so maybe it's a little confusing, though at the same time I don't think this is worth refactoring

Reply:
It's confusing but it works so fine for now